app/assets/stylesheets/settings.scss

```diff
@@ -30,7 +30,7 @@ $whitish-grey: #d8d8d8;
 $red-danger: #dd4b39;
 $google-color: #dd4b39;
 $success-color: #96B874;
-$viking-logo-text: #11343d;
+$advert-logo-text: #11343d;
```

Comment:
Let's remove the beginning `-` here please. Will need to be changed in `assets/stylesheets/components/advert.scss`